### ISSUE TITLE
feature request - pull the logs

### ISSUE THREAD

**Issue by digininja** [tautology0/ayfabtu#7]
With all that you are already grabbing, would it be possible to also grab the logs?

**Comment by tautology0** [comment]
Yes good idea - this may allow discover of more submissions - need to work out the best way to display them.

**Comment by digininja** [comment]
Have a look at:

```
git shortlog
```

And 

```
git shortlog +s
```

For some ideas.

**Comment by tautology0** [comment]
Basic commit added for now, still more logs to examine and optionally allow to add to discovery.
